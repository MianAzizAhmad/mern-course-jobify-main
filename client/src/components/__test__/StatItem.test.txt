import React from 'react';
import StatsItem from './../StatItem';
import "@testing-library/jest-dom/extend-expect"
import renderer from 'react-test-renderer';

describe('StateItem Test Case { ', ()=>{

    it(' Snap 1 }', ()=>{
        
        const tree = renderer.create(<StatsItem count="3" title="Snap 1" icon="Snap" color="white" bcg="dark"/>).toJSON();
        
        expect(tree).toMatchSnapshot();
    });

    it(' Snap 2 }', ()=>{
        const tree = renderer.create(<StatsItem ></StatsItem>).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it(' Snap 3 }', ()=>{
        
        const tree = renderer.create(<StatsItem count="0" title="Snap 3" icon="Snap" color="dark" bcg="white"></StatsItem>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});