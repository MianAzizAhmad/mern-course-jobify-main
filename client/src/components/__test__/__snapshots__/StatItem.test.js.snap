// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StateItem Test Case {   Snap 1 } 1`] = `
<header>
  <span
    className="count"
  >
    3
  </span>
  <span
    className="icon"
  >
    Snap
  </span>
</header>
`;

exports[`StateItem Test Case {   Snap 2 } 1`] = `
<header>
  <span
    className="count"
  />
  <span
    className="icon"
  />
</header>
`;

exports[`StateItem Test Case {   Snap 3 } 1`] = `
<header>
  <span
    className="count"
  >
    0
  </span>
  <span
    className="icon"
  >
    Snap
  </span>
</header>
`;
